src/main/config/config_streamer.h

```diff
@@ -23,32 +23,43 @@
 #include <stdint.h>
 #include <stdbool.h>
 
+#include "platform.h"
+
 // Streams data out to the EEPROM, padding to the write size as
 // needed, and updating the checksum as it goes.
 
 #if defined(CONFIG_IN_EXTERNAL_FLASH) || defined(CONFIG_IN_MEMORY_MAPPED_FLASH)
 #define CONFIG_STREAMER_BUFFER_SIZE 8 // Must not be greater than the smallest flash page size of all compiled-in flash devices.
-typedef uint32_t config_streamer_buffer_align_type_t;
+#define CONFIG_BUFFER_ALIGN_TYPE    uint32_t
```

Comment:
IMO his should be `CONFIG_BUFFER_TYPE` or similar - it is used both for alignment and to write buffer word (`*buffer`)